coordinator/src/services/translation/request/withdraw/withdraw.ts

```diff
@@ -29,10 +31,13 @@ export function createRecordTarget(identifier: fhir.Identifier): hl7V3.RecordTar
   return new hl7V3.RecordTargetReference(patient)
 }
 
-export function createAuthor(): hl7V3.AuthorPersonSds {
+export function createAuthor(headers: Hapi.Util.Dictionary<string>): hl7V3.AuthorPersonSds {
+  const sdsRoleProfileId = getSdsRoleProfileId(headers)
+  const sdsUserUniqueId = getSdsUserUniqueId(headers)
+
   const agentPersonSds = new hl7V3.AgentPersonSds()
-  agentPersonSds.id = new hl7V3.UnattendedSdsRoleProfileIdentifier()
```

Comment:
I think this might be the last reference to these dummy Unattended... classes, in which case we can get rid of them